File: app/src/main/java/com/dwett/habits/Migrations.java
package com.dwett.habits;

import androidx.sqlite.db.SupportSQLiteDatabase;
import androidx.room.migration.Migration;
import androidx.annotation.NonNull;

public class Migrations {
    public static final Migration MIGRATION_1_2 = new Migration(1, 2) {
        @Override
        public void migrate(@NonNull SupportSQLiteDatabase database) {
            database.execSQL("ALTER TABLE `habit` ADD COLUMN `archived_2` INTEGER" +
                    " NOT NULL DEFAULT `0`");
        }
    };
}

File: app/src/main/java/com/meenbeese/habitic/Migrations.kt
package com.meenbeese.habitic

import androidx.sqlite.db.SupportSQLiteDatabase
import androidx.room.migration.Migration

class Migrations {
    companion object {
        val MIGRATION_1_2: Migration = object : Migration(1, 2) {
            override fun migrate(database: SupportSQLiteDatabase) {
                database.execSQL(
                    "ALTER TABLE `habit` ADD COLUMN `archived_2` INTEGER" +
                            " NOT NULL DEFAULT `0`"
                )
            }
        }
    }
}